pkg/graveler/graveler.go

```diff
@@ -111,9 +108,22 @@ type MergeStrategy int
 const (
 	MergeStrategyNone MergeStrategy = iota
 	MergeStrategyDest
-	MergeStrategySource
+	MergeStrategySrc
+
+	MergeStrategyNoneStr     = "default"
+	MergeStrategyDestWinsStr = "dest-wins"
+	MergeStrategySrcWinsStr  = "source-wins"
+
+	MergeStrategyMetadataKey = "lakefs.merge.strategy"
```

Comment:
Nice!

I would like us to take the opportunity to announce that some metadata keys are reserved for lakefs, and to **document** this fact.  At the same time, we might prefer to use `_lakefs` or `%lakefs` or `.lakefs`, to help people acknowledge that it is ours.

Reply:
Modified to .lakefs.
Will open a separate issue for documentation